certbot/tests/compat/os_test.py

```diff
@@ -7,8 +7,12 @@
 class OsTest(unittest.TestCase):
     """Unit tests for os module."""
     def test_forbidden_methods(self):
+        # Checks for os module
         for method in ['chmod', 'chown', 'open', 'mkdir', 'makedirs', 'rename', 'replace']:
             self.assertRaises(RuntimeError, getattr(os, method))
+        # Checks for os.path module
+        for method in ['realpath']:
+            self.assertRaises(RuntimeError, getattr(os.path, method))
```

Comment:
If we don't make `compat.path` private, for consistency I think we should make a `path_test.py` file for this test and then just add a test to this file that `compat.os.path` is `compat.path`.

Reply:
Since I made the `path` module internal, this comment is not relevant anymore.